docs/setup.md

```diff
@@ -181,7 +181,7 @@ When installing on Ubuntu or Debian you can either download the `deb` package,
 [install manually or build from source](#installing-from-source) or use our APT repository.
 
 ```bash
-$ curl http://packages.gopass.pw/repos/gopass/gopass-archive-keyring.gpg | sudo tee /usr/share/keyrings/gopass-archive-keyring.gpg
```

Comment:
Actually the problem is that the keyring should be fetched over https, not http. Then curl wouldn't get a redirect and would succeed.

I think it's correct to fail in this case.

Reply:
Ah, that's why. Good catch.